pkg/object/ingresscontroller/translator.go

```diff
@@ -86,6 +86,7 @@ func (b *pipelineSpecBuilder) addProxy(endpoints []string) {
 		"kind":     proxy.Kind,
 		"name":     name,
 		"mainPool": pool,
```

Comment:
please also remove this line.

Reply:
which line? please tell me the line number

Reply:
L88, it is now useless.

Reply:
I removed this line, but it failed with validation error in pkg/filters/proxy/proxy.go:132. So I didn't remove this line.
```go
// Validate validates Spec.
func (s *Spec) Validate() error {
	numMainPool := 0
	for i, pool := range s.Pools {
		if pool.Filter == nil {
			numMainPool++
		}
		if err := pool.Validate(); err != nil {
			return fmt.Errorf("pool %d: %v", i, err)
		}
	}

	if numMainPool != 1 {
		return fmt.Errorf("one and only one mainPool is required")
	}

	if s.MirrorPool != nil {
		if s.MirrorPool.Filter == nil {
			return fmt.Errorf("filter of mirrorPool is required")
		}
		if s.MirrorPool.MemoryCache != nil {
			return fmt.Errorf("memoryCache must be empty in mirrorPool")
		}
	}

	return nil
}
```

Reply:
could you please paste the detailed validation error here? thanks.